packages/components/.eslintrc

```diff
@@ -3,6 +3,8 @@
 	"rules": {
 		"react/forbid-foreign-prop-types": 0,
 		"react/jsx-wrap-multilines": 0,
-		"no-console": [2, { "allow": ["warn", "error"] }]
+		"no-console": [2, { "allow": ["warn", "error"] }],
+		"react/state-in-constructor": 0,
+		"react/jsx-indent": 0
```

Comment:
TODO: move that to talend/scripts instead

- react/state-in-constructor: we don't really care if a state is created in constructor or no constructor and created as class attribute.
- react/jsx-indent: let prettier do its job

Reply:
I prefer the state created in the class attribute but i don't care that much as we don't tend to create new classes for components